applications/CompressiblePotentialFlowApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -12,12 +12,7 @@
 
```

Comment:
I would delete this file entirely since it is not used
Up to you

If you keep it then you should also change to `define_python`